### PR TITLE
Add a demo for reactor-grpc

### LINKED ISSUE #11: Add demo for Reactor

### PR DESCRIPTION
**Pull request by cbornet**
Fix #11

### CODE REVIEW

File: demos/ReactorChat/ReactorChat-Server/pom.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-logging</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
+            <version>4.2.0.RELEASE</version>
+        </dependency>
+        <!-- Force Reactor 3 instead of Reactor 2 which is pulled by the current Spring Boot-->
+        <dependency>
```
Comment (cbornet) [inline demos/ReactorChat/ReactorChat-Server/pom.xml]: Or maybe use Spring Boot 2.0.0.M6 ?
@rmichela WDYT ?

Comment (rmichela) [reply demos/ReactorChat/ReactorChat-Server/pom.xml]: I'm fine with either. Reactor 3 is the most exciting part. Is Reactor 3 backward compatible with Reactor 2?

Comment (cbornet) [reply demos/ReactorChat/ReactorChat-Server/pom.xml]: > Is Reactor 3 backward compatible with Reactor 2?

I don't think so : Flux used to be Stream, Mono used to be Promise, etc...
So the code generated by the plugin only works with Reactor 3. Maybe that's worth mentioning.

Comment (cbornet) [reply demos/ReactorChat/ReactorChat-Server/pom.xml]: Note that Reactor 2 is quite old and legacy (last release in April 2016)

**Pull request merged**